network-store-client/src/main/java/com/powsybl/network/store/client/RestClientImpl.java

```diff
@@ -103,6 +103,19 @@ public <T extends IdentifiableAttributes> Optional<Resource<T>> getOne(String ta
         }
     }
 
+    @Override
+    public Optional<ExtensionAttributes> getOneExtensionAttributes(String url, Object... uriVariables) {
+        ResponseEntity<ExtensionAttributes> response = restTemplate.exchange(url, HttpMethod.GET, null, new ParameterizedTypeReference<>() { }, uriVariables);
```

Comment:
should be wrapped in TopLevelDocument

Reply:
ExtensionAttributes are not IdentifiableAttributes so it can't we wrapped in a TopLevelDocument as is that's why I went that way.
I also thought about making TopLevelDocument generic (https://github.com/powsybl/powsybl-network-store/pull/423) but if we want to go that way we may want to use the method https://github.com/powsybl/powsybl-network-store/blob/96b8ad68f72ddf6559b3f8c31e8114fb3478b01e/network-store-client/src/main/java/com/powsybl/network/store/client/RestClient.java#L33 that would use a TopLevelDocument instead of getOne() getAll(). 
In the current implementation when we need to retrieve objects that can't be wrapped in a TopLevelDocument we use directly the get() method https://github.com/powsybl/powsybl-network-store/blob/e5d30f4a1460fe320055302ddbd40a1a9d7ab77a/network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java#L209 so I followed this for ExtensionAttributes.
We could potentially also wrap the maps in TopLevelDocument if we want to with a generic TopLevelDocument.
We can also have a specific TopLevelDocument for ExtensionAttributes?